File: src/main/java/cookiedragon/luchadora/util/ISerializable.java
package cookiedragon.luchadora.util;

import com.google.gson.JsonObject;

/**
 * @author cookiedragon234 15/Dec/2019
 */
public interface ISerializable
{
	JsonObject addToObject(JsonObject jsonObject);
	void retrieveFromObject(JsonObject jsonObject);
}

File: src/main/kotlin/cookiedragon/luchadora/util/ISerializable.kt
package cookiedragon.luchadora.util

import com.google.gson.JsonObject

/**
 * @author cookiedragon234 17/Feb/2020
 */
interface ISerializable {
	fun addToObject(jsonObject: JsonObject): JsonObject
	fun retrieveFromObject(jsonObject: JsonObject)
}
